.circleci/config.yml

```diff
@@ -54,7 +54,7 @@ jobs:
             flake8 dash setup.py && flake8 --ignore=E123,E126,E501,E722,E731,F401,F841,W503,W504 --exclude=metadata_test.py tests
             pylint dash setup.py --rcfile=$PYLINTRC
             pylint tests/unit tests/integration/devtools tests/integration/renderer tests/integration/dash_assets -d all -e C0410,C0411,C0412,C0413,W0109
-            cd dash-renderer && npm install --ignore-scripts
+            cd dash-renderer && npm ci
```

Comment:
Does `npm ci` just always ignore scripts? That seems implied, but I can't find it documented anywhere...

Reply:
`--ignore-scripts` prevents `prepublish` and `prepare` from being run, these commands are not used anymore in any of the core packages